docs/DeveloperGuide.md

```diff
@@ -1,34 +1,845 @@
-# Developer Guide
+By: `Team M16-1` Since: `Jan 2020` License: `MIT`
 
-## Design & Implementation
+Content
+- [Setting up](#1-setting-up)
```

Comment:
Section 1 and section 3 both mentions setting up, could they be mentioned under the same section instead?